### ISSUE TITLE
Exploring Kialo.com's Interactive Debate Tree as a Discussion Tool for Proposals

### ISSUE THREAD

**Issue by venezuela01** [loki-project/loki-improvement-protocols#37]
When it comes to examining large proposals such as [ORC-8 The Session Network Token](https://github.com/oxen-io/oxen-improvement-proposals/issues/36), it's crucial to conduct a thorough assessment of the pros and cons associated with each argument. Additionally, every argument may potentially branch off into related subtopics. For instance, [A theoretical estimation of Oxen chain's storage size vs Session message storage requirement](https://github.com/oxen-io/oxen-storage-server/issues/480) is a subtopic of [ORC-8 The Session Network Token](https://github.com/oxen-io/oxen-improvement-proposals/issues/36).

Traditional linear discussion threads fall short when it comes to facilitating complex, large-scale, and in-depth discussions. They can make the process of providing and receiving feedback cumbersome, as valuable insights may be mixed with noise and repeated opinions.

Addressing this issue, the [Kialo project](https://www.kialo.com/) was created to introduce an interactive tree structure that can neatly organize intricate arguments and foster effective critical thinking.

I invite all community members to explore Kialo with some random topics, such as [Minimum wage laws are bad](https://www.kialo.com/minimum-wage-laws-are-bad-59328?path=59328.0~59328.3). Consider its potential applicability for Oxen-related proposal discussions, and please share your experiences or thoughts as comments on this issue.

Here are screenshots of what an interactive debate tree looks like:

<img width="540" alt="interactive debate tree 1" src="https://github.com/oxen-io/oxen-improvement-proposals/assets/138533453/8e769e2d-621c-4e12-a46c-83796fc0059b">

<img width="360" alt="interactive debate tree 2" src="https://github.com/oxen-io/oxen-improvement-proposals/assets/138533453/00dba9b7-ef27-46a5-b8ef-14a6fbfe917d">

Given sufficient interest, I'll volunteer to manually mirror and sync some topics, subtopics, and opinions from [ORC-8 The Session Network Token](https://github.com/oxen-io/oxen-improvement-proposals/issues/36) to Kialo. After some experiments we can evaluate the tangible improvements this brings to our discussions.

Initially, it won't be necessary for everyone to register with Kialo, as I plan to manually forward discussions from GitHub to Kialo. However, if Kialo proves to be of significant value, I'll encourage a broader participation.

Please feel free to beat me in speed in case if you also like the idea.

**Comment by KeeJef** [comment]
This would have been great at the time of announcement, however i think it could be too late now, might be better for future community proposals?

**Comment by venezuela01** [comment]
Yes, it might be a bit late this time. I can volunteer to manually collect and mirror discussions from Github/Telegram/Session to Kialo this time, so we can evaluate the efficiency of the tool, and next time we can consider promoting it to a wider range of audience.

**Comment by KeeJef** [comment]
Sounds good 👍

**Comment by venezuela01** [comment]
Update:

After evaluating Kialo and comparing it with some other tools, I found it is a little bit too broad for the challenges we face.

For example, Kialo does not encourage cost-benefit analysis. Most discussions on Kialo are in a 'cons vs pros' style. The key difference is that 'cost and benefit' is 'weighted,' where we are supposed to sum up the cost (negative number) and benefit (positive number) and make a conclusion on whether a strategy leads to a good ROI. However, for 'cons vs pros' analysis, it's less obvious which direction one should go, especially when a large number of both cons and pros are raised simultaneously by supporters and opponents. 

Instead, I discovered a few tools on Leanfoundry.com that usually offer a more specific and productive discussion framework for startup topics. I'll close this proposal and leave some remaining tooling discussion in [Embrace Lean Startup Methodology for the Session Project](https://github.com/oxen-io/oxen-improvement-proposals/issues/67) when appropriate.
